azerothcore/azerothcore-wotlk
# fix(DB/Smartscript): Private Hendel should despawn on player dead/reset

## Changes Proposed:
-  Fixed issues with the smartscript locking up making people unable to complete the quest till server restart.
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7390
- Closes https://github.com/chromiecraft/chromiecraft/issues/1683

## SOURCE:


## Tests Performed:

- Tested
- 


## How to Test the Changes:


1. .q add 1266
2. .q ender 1266
3. die from hendel see that they despawn and respawn after 60 seconds
4. make sure you can complete the quest normally

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
